tests/WE2E/machine_suites/comprehensive.cheyenne

```diff
@@ -1,3 +1,4 @@
+2020_CAD
```

Comment:
Ah I had forgotten that Cheyenne can't automatically get AWS data. I had just noticed that test was only added to the "Orion" comprehensive list when it shouldn't be restricted to just that machine. I have made that update.

Regarding the new `custom_ESGgrid_Great_Lakes_snow_8km` test and what platforms it runs on, the idea behind the "comprehensive" test suite is that *all* tests that can run on a given platform should be included. So unless a test is known to fail, it should go in the `comprehensive` file. If Gaea can't run HPSS tests, then it should have its own `comprehensive.gaea` file that omits tests requiring HPSS access. I had just assumed Gaea had HPSS access since it didn't have its own file yet (I don't have access to this machine).

Reply:
It makes sense to add the new `custom_ESGgrid_Great_Lakes_snow_8km` to the `comprehensive` test file.  Technically, the SRW App doesn't build or run on Gaea currently, so including this test in the `comprehensive` test file will work for now.  Once the SRW App is once again able to build and run on Gaea, I will try running the `comprehensive` test file on that machine and create a new `comprehensive.gaea` file that omits the tests requiring HPSS access from that suite if those tests fail.